macros/path_transformations/path_transformation.sql

```diff
@@ -0,0 +1,47 @@
+{% macro path_transformation(transformation_type, transform_param) %}
+  {{ return(adapter.dispatch('path_transformation', 'snowplow_fractribution')(transformation_type, transform_param)) }}
+{% endmacro %}
+
+-- only used for integration tests
+{% macro default__path_transformation(transformation_type, transform_param) %}
+
```

Comment:
I don't think so. The transform_path macro will use the same code as this macro for Snowflake/Bigquery. It is to avoid the issue we faced when trying to generate a macro call with a parameter.